vision_stack/camera/bottomcam/include/bottomcam/bottomcam.h

```diff
@@ -0,0 +1,34 @@
+#include <ros/ros.h>
+#include <image_transport/image_transport.h>
+#include <cv_bridge/cv_bridge.h>
+#include <sensor_msgs/image_encodings.h>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include <std_msgs/String.h>
+
+#include <resources/topicHeader.h>
+
+class Camera
+{
+public:
+    Camera(int camNo=0);
+    cv::VideoCapture cam;
+    int cameraNo;
```

Comment:
When you provide the get/set methods, I think this variable access should be made private.